### DISCUSSION TITLE
How to use wire:modal on the  backend (Livewire controller)

### DISCUSSION THREAD

**Discussion by nenadsijan** [wire-elements/modal#457] [Q&A]
Dear @PhiloNL 
This solution `<button wire:modal="edit-user, @js(['user' => 1)">Open</button>` works perfectly on the blade page, but how to use this `wire:modal` call from Livewire controller ?
Correctly for this scenario when i need to open new modal from backend.
`$this->dispatch('modal.open', component: 'modals.edit-user', arguments: ['user' => 1]);`
Thanks in advance
Nenad

**Comment by PhiloNL** [comment]
The back-end code remains unchanged, so the example you provided is the correct way.

**Comment by SfaygaTheKing** [comment]
i thought about this as a solution to routing my modal. i added a function to my object controller that return the index page and defined it in web.php but i don't know how to dispatch the modal from the controller or to give like a token to the frontend to open a certain modal.
